packages/@lightningjs/ui-components/src/components/Key/Key.d.ts

```diff
@@ -27,17 +28,85 @@ type SizeObject = {
   xxl: number;
 };
 
-export type KeyStyle = ButtonStyle & {
+type KeyStyle = ButtonStyle & {
   h: number;
   sizes: SizeObject;
   iconW: number;
   iconH: number;
 };
 
-export default class Key extends Button {
+declare namespace Key {
+  export interface TemplateSpec extends Button.TemplateSpec {
+    /**
+     * url for icon
+     */
+    icon?: string;
+
+    /**
+     * it allows custom handling of the "Enter" key press behavior in the Key component, overriding the default $onSoftKey and $toggleKeyboard events.
+     */
+    onEnter?: (key: string, toggle: boolean) => void;
+
+    /**
+     * width of the Key
+     */
+    size?: string;
+
+    /**
+     * used to toggle `$toggleKeyboard`
+     */
+    toggle?: boolean;
+
+    style?: StylePartial<KeyStyle>;
+  }
+  export interface TypeConfig extends lng.Component.TypeConfig {
+    SignalMapType: SignalMap;
+  }
+  /**
+   * Signal map interface for Key component
+   */
+  export type SignalMap = {
```

Comment:
It doesn't seem like Key has any signals?

Reply:
Sure, I will remove that now then, Thanks!